acceptance/testdata/analyzer/analyze-image/Dockerfile.windows

```diff
@@ -8,3 +8,5 @@ WORKDIR /layers
 ENV CNB_USER_ID=1
 
 ENV CNB_GROUP_ID=1
+
+ENV CNB_PLATFORM_API=${cnb_platform_api}
```

Comment:
Hmm... I actually don't think we need this at all. Since we are always passing in `"--env", "CNB_PLATFORM_API="+apiString`. Actually what we might really want is to change this to `ENV CNB_PLATFORM_API=invalid-api` (or something) so that if we forget to pass in the platform api we will get an obvious failure.